hw20/js/script.js

```diff
@@ -0,0 +1,27 @@
+function manipulateTheList() {
+  const elementsOfDrinks = document.querySelectorAll("#drinks-list li");
+  const arrayOfDrinks = Array.from(elementsOfDrinks).map(
+    (li) => li.textContent
+  );
+  const uniqueSortedArray = sortUniqueArray(arrayOfDrinks);
+  console.log(uniqueArray);
+
+  const listOfDrinks = document.querySelector("#drinks-list");
+  listOfDrinks.innerHTML = "";
+
+  uniqueSortedArray.forEach((content) => {
+    const elementOfList = document.createElement("li");
+    elementOfList.textContent = content;
+    listOfDrinks.append(elementOfList);
+  });
+}
+
+function sortUniqueArray(array) {
+  let sortedArray = [];
+  array.filter((element) => {
+    if (!sortedArray.includes(element)) sortedArray.push(element);
+  });
+  return sortedArray.sort();
+}
```

Comment:
this function could be shortened as
```javascript
function sortUniqueArray(array) {
  return Array.from(new Set(array)).sort();
}
```
read about global object [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)